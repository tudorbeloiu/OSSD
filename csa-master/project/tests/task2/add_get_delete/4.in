40
1
5
162
5069
88
4423
146
3085
56
6170
107
4229
1
8
25
1521
168
2448
163
4857
113
5812
17
276
122
8198
172
8806
33
2070
1
4
173
716
188
534
87
7467
10
7359
3
25
3
122
2
88
2
146
3
163
3
10
2
17
2
173
1
7
45
4339
84
803
60
3953
67
4037
141
5238
97
8738
239
8221
3
56
3
45
3
67
3
162
2
141
2
88
2
88
2
107
2
88
3
97
2
168
3
172
2
60
2
188
2
87
1
10
155
5034
182
1262
115
8089
245
6607
2
7947
156
8277
201
5419
121
4695
248
4815
252
3186
2
84
2
173
2
173
1
7
24
3110
221
5313
13
304
44
1727
176
849
39
7388
29
5296
3
201
2
155
3
188
2
115
1
5
253
1496
100
4561
40
509
11
6840
201
3644
2
182
1
7
27
3135
215
7545
123
5293
99
6040
232
5708
199
6364
77
8030
3
84
