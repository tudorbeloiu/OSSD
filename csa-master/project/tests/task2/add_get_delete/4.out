162: ((0, 0), (0, 633))
88: ((1, 0), (1, 552))
146: ((0, 634), (0, 1019))
56: ((2, 0), (2, 771))
107: ((3, 0), (3, 528))
25: ((1, 553), (1, 743))
168: ((3, 529), (3, 834))
163: ((4, 0), (4, 607))
113: ((5, 0), (5, 726))
17: ((1, 744), (1, 778))
122: ((0, 0), (0, 0))
172: ((0, 0), (0, 0))
33: ((4, 608), (4, 866))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
87: ((6, 0), (6, 933))
10: ((7, 0), (7, 919))
162: ((0, 0), (0, 633))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
56: ((2, 0), (2, 771))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
163: ((4, 0), (4, 607))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
10: ((7, 0), (7, 919))
162: ((0, 0), (0, 633))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
56: ((2, 0), (2, 771))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
163: ((4, 0), (4, 607))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
10: ((7, 0), (7, 919))
((1, 0), (1, 552))
((0, 634), (0, 1019))
162: ((0, 0), (0, 633))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
56: ((2, 0), (2, 771))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
10: ((7, 0), (7, 919))
162: ((0, 0), (0, 633))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
56: ((2, 0), (2, 771))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
((1, 744), (1, 778))
((1, 779), (1, 868))
45: ((4, 0), (4, 542))
84: ((1, 553), (1, 653))
60: ((7, 0), (7, 494))
67: ((7, 495), (7, 999))
141: ((8, 0), (8, 654))
97: ((0, 0), (0, 0))
239: ((0, 0), (0, 0))
162: ((0, 0), (0, 633))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
45: ((4, 0), (4, 542))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
67: ((7, 495), (7, 999))
141: ((8, 0), (8, 654))
162: ((0, 0), (0, 633))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
67: ((7, 495), (7, 999))
141: ((8, 0), (8, 654))
162: ((0, 0), (0, 633))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
141: ((8, 0), (8, 654))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
141: ((8, 0), (8, 654))
((8, 0), (8, 654))
((1, 0), (1, 552))
((1, 0), (1, 552))
((3, 0), (3, 528))
((1, 0), (1, 552))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
141: ((8, 0), (8, 654))
((3, 529), (3, 834))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
141: ((8, 0), (8, 654))
((7, 0), (7, 494))
((1, 869), (1, 935))
((6, 0), (6, 933))
155: ((0, 0), (0, 629))
182: ((2, 0), (2, 157))
115: ((9, 0), (9, 1011))
245: ((2, 158), (2, 983))
2: ((10, 0), (10, 993))
156: ((0, 0), (0, 0))
201: ((11, 0), (11, 677))
121: ((4, 0), (4, 586))
248: ((12, 0), (12, 601))
252: ((7, 495), (7, 893))
((1, 553), (1, 653))
((1, 779), (1, 868))
((1, 779), (1, 868))
24: ((12, 602), (12, 990))
221: ((13, 0), (13, 664))
13: ((1, 654), (1, 691))
44: ((5, 727), (5, 942))
176: ((3, 835), (3, 941))
39: ((14, 0), (14, 923))
29: ((15, 0), (15, 661))
155: ((0, 0), (0, 629))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
13: ((1, 654), (1, 691))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
188: ((1, 869), (1, 935))
182: ((2, 0), (2, 157))
245: ((2, 158), (2, 983))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
176: ((3, 835), (3, 941))
121: ((4, 0), (4, 586))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
44: ((5, 727), (5, 942))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
252: ((7, 495), (7, 893))
141: ((8, 0), (8, 654))
115: ((9, 0), (9, 1011))
2: ((10, 0), (10, 993))
248: ((12, 0), (12, 601))
24: ((12, 602), (12, 990))
221: ((13, 0), (13, 664))
39: ((14, 0), (14, 923))
29: ((15, 0), (15, 661))
((0, 0), (0, 629))
155: ((0, 0), (0, 629))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
84: ((1, 553), (1, 653))
13: ((1, 654), (1, 691))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
182: ((2, 0), (2, 157))
245: ((2, 158), (2, 983))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
176: ((3, 835), (3, 941))
121: ((4, 0), (4, 586))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
44: ((5, 727), (5, 942))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
252: ((7, 495), (7, 893))
141: ((8, 0), (8, 654))
115: ((9, 0), (9, 1011))
2: ((10, 0), (10, 993))
248: ((12, 0), (12, 601))
24: ((12, 602), (12, 990))
221: ((13, 0), (13, 664))
39: ((14, 0), (14, 923))
29: ((15, 0), (15, 661))
((9, 0), (9, 1011))
253: ((8, 655), (8, 841))
100: ((11, 0), (11, 570))
40: ((1, 869), (1, 932))
11: ((16, 0), (16, 854))
201: ((17, 0), (17, 455))
((2, 0), (2, 157))
27: ((11, 571), (11, 962))
215: ((18, 0), (18, 943))
123: ((19, 0), (19, 661))
99: ((20, 0), (20, 754))
232: ((21, 0), (21, 713))
199: ((22, 0), (22, 795))
77: ((23, 0), (23, 1003))
155: ((0, 0), (0, 629))
146: ((0, 634), (0, 1019))
88: ((1, 0), (1, 552))
13: ((1, 654), (1, 691))
17: ((1, 744), (1, 778))
173: ((1, 779), (1, 868))
40: ((1, 869), (1, 932))
182: ((2, 0), (2, 157))
245: ((2, 158), (2, 983))
107: ((3, 0), (3, 528))
168: ((3, 529), (3, 834))
176: ((3, 835), (3, 941))
121: ((4, 0), (4, 586))
33: ((4, 608), (4, 866))
113: ((5, 0), (5, 726))
44: ((5, 727), (5, 942))
87: ((6, 0), (6, 933))
60: ((7, 0), (7, 494))
252: ((7, 495), (7, 893))
141: ((8, 0), (8, 654))
253: ((8, 655), (8, 841))
115: ((9, 0), (9, 1011))
2: ((10, 0), (10, 993))
100: ((11, 0), (11, 570))
27: ((11, 571), (11, 962))
248: ((12, 0), (12, 601))
24: ((12, 602), (12, 990))
221: ((13, 0), (13, 664))
39: ((14, 0), (14, 923))
29: ((15, 0), (15, 661))
11: ((16, 0), (16, 854))
201: ((17, 0), (17, 455))
215: ((18, 0), (18, 943))
123: ((19, 0), (19, 661))
99: ((20, 0), (20, 754))
232: ((21, 0), (21, 713))
199: ((22, 0), (22, 795))
77: ((23, 0), (23, 1003))
